client/src/reducers/undoableConfig.js

```diff
@@ -229,7 +240,7 @@ function shallowArrayEq(arrA, arrB) {
 }
 
 /* configuration for the undoable meta reducer */
-const debug = false; // set truish for undoble debugging
+const debug = false; // set truish for undoble debugging. Higher number is more verbose.
```

Comment:
Comment does not reflect line of code

Reply:
sure it does!  but I'll make it even more obvious :-)